File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int val = x;
        int car = 0;
        while(x>0){
            int temp = x%10;
            car = car * 10 + temp;
            x= x/10;
        }
        return (val == car);
        
    }
}

File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long long int rev =0 , n=x;
        if(x<0){
            return false;
        }
        while(x!=0){
            rev *=  10;
            rev += (x%10);
            x  /=  10;
        }
        if(n==rev) return 1;
        else return 0;
    }
};